.claude/agents/function-implementer.md

```
@@ -0,0 +1,110 @@
+
+```
+1. Write TDD tests in tests/ directory
+2. Implement function to pass tests
+3. Run: pnpm test
+4. Verify: pnpm test:coverage
+5. If .kiro/spec/<spec-name>/ exists:
+   a. Write PBT tests in .kiro/spec/<spec-name>/<function-name>.pbt.test.ts
+   b. Run: pnpm test:pbt
+   c. Fix any property violations
+   d. Verify all properties pass
+6. Mark implementation as complete
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)